src/Akeneo/UserManagement/Component/Model/User.php

```diff
@@ -1123,6 +1123,51 @@ public function getProperty(string $propertyName)
         return $this->properties[$propertyName] ?? null;
     }
 
+    public function duplicate(): UserInterface
+    {
+        $duplicated = new self();
```

Comment:
```suggestion
        $duplicated = new static();
```
(in case this class is extended, but the `duplicate()` method is not)